src/Common/src/System/Security/Cryptography/DSACng.cs

```diff
@@ -14,11 +15,13 @@ internal static partial class DSAImplementation
 #endif
         public sealed partial class DSACng : DSA
         {
+            private static readonly int s_defaultKeySize = Supports2048KeySize() ? 2048 : 1024;
```

Comment:
For ECDsa the legal key sizes are not complete, but at least are all always valid.
The common DSA tests only look at the first two legal sizes so we are good on tests (doing max key size would cause the tests to run for a long time)